### PR TITLE
Create 2021-12-16-incubator-lesson-spotlight-jekyll-github.md

### PR DESCRIPTION
**Pull request by kmomar**

### CODE REVIEW

File: _posts/2021/12/2021-12-16-incubator-lesson-spotlight-jekyll-github.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+authors: ["Renato Alves", "Anne Fouilloux", "Toby Hodges", "Aleksandra Nenadic", "Sarah Stevens"]
+teaser: "Get involved in the community’s development of a new lesson to teach Jekyll and GitHub."
+title: "Incubator Lesson Spotlight: Building Websites with Jekyll & GitHub Pages"
+date: 2021-12-16
+time: "09:00:00"
+tags: ["Carpentries Incubator Lesson Spotlight", "Incubator", "Community"]
+---
```
Comment (MCMaurer) [inline _posts/2021/12/2021-12-16-incubator-lesson-spotlight-jekyll-github.md]: Just need to add the following line after the `tags` line:

```
lesson_title: “Building Websites with Jekyll & GitHub Pages”

**Comment by MCMaurer** [comment]
Hi @unode, @annefou, @anenadic, and @sstevens2. We've drafted up a blog post for your submission to the Carpentries Incubator Lesson Spotlight- if you want to give it a quick review and note any changes or approve it, that would be great! We are hoping to get the post out before people disperse for holidays, but if that timeline is too short to sign off on it, we can certainly push until January. Thanks!

**Review by unode — APPROVED** [review]
Looks good to me!

**Review by sstevens2 — APPROVED** [review]
Thanks for writing this!  Looks great! Hope we get some new folks teaching it!

**Comment by anenadic** [comment]
Thank you @kmomar! 🎉

**Pull request merged**